packages/aws-cdk/lib/api/hotswap/lambda-functions.ts

```diff
@@ -120,6 +122,15 @@ async function isLambdaFunctionCodeOnlyChange(
               foundCodeDifference = true;
               s3Key = await evaluateCfnTemplate.evaluateCfnExpression(updatedProp.newValue[newPropName]);
               break;
+            case 'ZipFile':
+              foundCodeDifference = true;
+              // We must create a zip package containing a file with the inline code
+              const rawZipFile = await evaluateCfnTemplate.evaluateCfnExpression(updatedProp.newValue[newPropName]);
```

Comment:
I'm not in love with the name of this variable - this is not really the ZIP file here, this is just the function's code (I understand it will _become_ a ZIP file later, but that's a different story 😛).

How about we call this `functionCode` instead?